packages/typespec-bump-deps/src/cli/cli.ts

```diff
@@ -27,7 +27,7 @@ async function main() {
   for (const packageJsonPath of packageJsonPaths) {
     const content = await readFile(packageJsonPath);
     const packageJson = JSON.parse(content.toString());
-    const depTypes = ["dependencies", "devDependencies", "peerDependencies"];
+    const depTypes = ["dependencies", "devDependencies", "peerDependencies", "overrides"];
```

Comment:
Can we add the `overrides` section in package.json in this script if absent?

Reply:
I think we can do this, though probably with a bit more discussion with Isabella & Timothee on parameter.

I kind of think `typespec-bump-deps --add-rush-overrides <file-path>` may do.
Though you will likely need to call it twice, one on `package.json`s to modify the deps, then call it on `pnpm-config.json` with that option to add overrides.

For npm it maybe `typespec-bump-deps --add-npm-overrides <file-path>`. (I assume Java and .NET may use this)

Let us continue this discussion when we are decided that JS codegen would indeed use the overrides.

Reply:
No, I don't need to call twice to upgrade them. I only need to call `typespec-bump-deps --add-rush-overrides <file-path>` and Rush would manage all rest of changes for me.

Reply:
`typespec-bump-deps --add-rush-overrides <file-path>` would be good to me. I would expect the following logic if enable `--add-rush-overrides` option:

1. Check if existing section `globalOverrides`; if yes, step 2; if no, adding one section;
2. For every `knownPackages` override its latest dev into `globalOverrides`